site/layouts/shortcodes/example.html

```diff
@@ -23,7 +25,7 @@
 
 {{- if eq $show_markup true -}}
   <div class="bd-edit">
-    <button type="button" class="btn-edit text-nowrap" title="Try it on StackBlitz">Try it</button>
+    <button type="button" class="btn-edit text-nowrap"{{ with $javascript }} data-js-file="{{ readFile (print "site/assets/js/" $javascript) }}"{{ end }} title="Try it on StackBlitz">Try it</button>
```

Comment:
I'd try to move the readfile call to a variable. It might be more readable. Also, might be worth using `path.join` like we do elsewhere.

Reply:
Modified in https://github.com/twbs/bootstrap/pull/36127/commits/bce7fb4993de6a6da813382e8c5bb3abbcf1e743. Used an intermediate var but maybe there's a way without it.